modules/comment/comment.service.js

```diff
@@ -79,9 +94,7 @@ class CommentsService {
       },
       { new: true }
     );
-
-    if (rateToAdd) return rateToAdd;
-    throw new Error(RATE_FOR_NOT_EXISTING_PRODUCT);
```

Comment:
Why you remove here error handling?

Reply:
i removed error handling from here because it is moved to the beginning of the function